app/models/filters/filter_base.rb

```diff
@@ -100,10 +102,12 @@ def update(filters) # rubocop:disable Metrics/CyclomaticComplexity, Metrics/Perc
         self.hoh_only = filter_hoh
       end
       self.project_type_codes = Array.wrap(filters.dig(:project_type_codes))&.reject(&:blank?).presence || project_type_codes
+      self.uses_default_project_type_codes = filters.dig(:project_type_codes).nil?
```

Comment:
Should we be able to pass in `FilterBase.new(uses_default_project_type_codes: false)` if so, we would need an additional clause here.

Reply:
I don't think so? Ithink being able to set it could create internal consistency problems.

Reply:
`self.uses_default_project_type_codes = ! filters.key?(:project_type_codes)`